app/controllers/stories_controller.rb

```diff
@@ -268,6 +268,7 @@ def assign_article_show_variables
     end
 
     @comments_to_show_count = @article.cached_tag_list_array.include?("discuss") ? 50 : 30
+    @comments_to_show_count = 15 unless user_signed_in?
```

Comment:
@jaw6 I think we do mean to override the statement above if the user is signed out, but I just wanted to double chcek?

Reply:
@Ridhwana that was how I understood [this request](https://github.com/forem/forem/issues/18490):

> Change "comments to show count" from its current number in the controller to 15 in all contexts for non-signed-in users (regardless of tags)

However, it might be good if @pkfrank or @benhalpern would like to confirm?

Reply:
Yes, this looks good to me.